app/views/activitypub/types/announce.activitystreams2.rabl

```diff
@@ -0,0 +1,3 @@
+extends 'activitypub/intransient.activitystreams2.rabl'
+
+node(:type) { 'Announce' }
```

Comment:
It's a little odd, but I did some more reading and it looks like Announce is the canonical AP way to do this: https://github.com/w3c/activitypub/issues/186

That said, it's not a bad idea to at least change the name of the activity to say "share."